### PR TITLE
Card Labels [#122] 

### LINKED ISSUE #122: Label Card Option
- Add option to label cards with player name when they are on the canvas so when they are moved/resorted you can track who's is who's.

### PR DESCRIPTION
**Pull request by JPMeehan**
- Added field on Cards configuration to set primary owner
- Added setting to toggle whether to show the card owners or not

Closes #122

### CODE REVIEW

File: lang/en.json [hunk 1]
```diff
@@ -73,7 +73,27 @@
...
+
+      "PrimaryOwner": "Primary Owner"
+    },
+    "Settings": {
+      "ShowNames": {
+        "Label": "Show Card Owner Name",
+        "Hint": "Display the owner of each card on the canvas"
+      },
+      "OwnerFontSize": {
+        "Label":"Font Size",
```
Comment (krbz999) [inline lang/en.json:86]:
```suggestion
        "Label": "Font Size",
```

File: src/module/canvas/CanvasCard.mjs [hunk 2]
```diff
@@ -131,6 +133,51 @@ export default class CanvasCard extends foundry.abstract.DataModel {
    */
   static flagProps = ["x", "y", "elevation", "sort", "rotation", "hidden", "locked", "flipped"];
 
+  static registerSettings() {
+    game.settings.register(MODULE_ID, "showOwner", {
+      name: "CCM.Settings.ShowNames.Label",
+      hint: "CCM.Settings.ShowNames.Hint",
+      scope: "client",
+      config: true,
+      type: new foundry.data.fields.BooleanField(),
```
Comment (krbz999) [inline src/module/canvas/CanvasCard.mjs:142]:
```suggestion
      type: new foundry.data.fields.BooleanField(),
      initial: true,
```

**Review by krbz999 — APPROVED** [review]
Seems good. We should reduce the font size on Cards objects (the Card objects' font size looks fine).

This is the default 160:
image


Git had a baby on your rebase, I think.

**Comment by JPMeehan** [comment]
The specific issue is whether or not there's a normal card texture; both of these are stacks but one has a proper back image.

image

**Comment by JPMeehan** [comment]
Going to merge and leave for UX review

**Pull request merged**